#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/manpage-hyphen-fix.patch.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Src/wmfrog.1: hyphen fix

@DPATCH@

diff --git a/Src/wmfrog.1 b/Src/wmfrog.1
index 36dc740..c0173e2 100644
--- a/Src/wmfrog.1
+++ b/Src/wmfrog.1
@@ -3,7 +3,7 @@
 .SH NAME
 wmfrog \- wmfrog
 .SH SYNOPSIS
-wmrog \fB\-s\fR <StationID> [-h] [-w <max wind>] [-t <time offset>] [-tmp <temp folder>] [-l <label>] [-m] [-delay <Time in Minutes>]
+wmrog \fB\\-s\fR <StationID> [\-h] [\-w <max wind>] [\-t <time offset>] [\-tmp <temp folder>] [\-l <label>] [\-m] [\-delay <Time in Minutes>]
 .SH DESCRIPTION
 \fB\-station\fR \fB\-s\fR <METAR StationID> 
 .IP
@@ -23,7 +23,7 @@ Set the maximum wind in KNOTS (for percentage of max / scaling.) default is 50 K
 .TP
 \fB\-o\fR <OFFSET>
 .IP
-Time offset of location from GMT (-8 for california, -1 for France etc...)
+Time offset of location from GMT (\-8 for california, \-1 for France etc...)
 .TP
 \fB\-tmp\fR <PATH>
 .IP
